en/docs/guides/identity-federation/claims-idp.md

```diff
@@ -34,6 +34,17 @@ To use local claim dialects:
 
 To define custom claim dialects:
 
+!!! note "Before you begin"
```

Comment:
@deshankoswatte I added a commit here. Could you please check?

Reply:
@nilminiwso2 it looks good to me.